webservices/filters.py

```diff
@@ -31,10 +40,10 @@ def filter_match(query, kwargs, fields):
 
 def filter_multi(query, kwargs, fields):
     for key, column in fields:
-        if kwargs.get(key):
-            # handle combination exclude/include lists
-            exclude_list = [parse_exclude_arg(value) for value in kwargs[key] if is_exclude_arg(value)]
-            include_list = [value for value in kwargs[key] if not is_exclude_arg(value)]
+        # if kwargs.get(key):
```

Comment:
I think we want to still check for `if kwargs.get(key):` here. If not, we need to fix the indentation. Also, we need two new lines between functions here.